providers/ruby.rb

```diff
@@ -81,8 +81,6 @@ def perform_install
 
     Chef::Log.debug("#{new_resource} build time was " \
       "#{(Time.now - install_start) / 60.0} minutes")
-
-    new_resource.updated_by_last_action(true)
```

Comment:
I removed one occurrence of [FC085](http://www.foodcritic.io/#FC085) but there is one more in [this file](https://github.com/sous-chefs/ruby_rbenv/blob/56b6ed3af0362d72ab1f6593ace45dcece74630e/libraries/chef_rbenv_script_helpers.rb#L59-L62) which gets called [a few times](https://github.com/sous-chefs/ruby_rbenv/search?utf8=✓&q=set_updated&type=) but I'm not sure if I can just remove it or if it's been replaced by something different. I saw [other cookbooks](https://github.com/chef-cookbooks/aws/commit/7c3ed4f5178e736e2db245155130928b66047314) just [removed it](https://github.com/dhoer/chef-nssm/commit/5de5e61c877abec9acab490e60baf5658ec1e807) but I haven't found any cookbook using it with something like `set_updated` in this one.